src/app/App.tsx

```diff
@@ -101,6 +102,7 @@ function App() {
                 <Route exact path="/people/:id">
                   <PersonPage />
                 </Route>
+                <Route component={ErrorPage} />
```

Comment:
Add a `<Route path="*" component={ErrorPage} />` as recommended by the [react-router doc](https://v5.reactrouter.com/web/example/no-match).